cmd/controller/main.go

```diff
@@ -96,19 +97,24 @@ func startPkgController(stopCh <-chan struct{}, cfg *rest.Config, logger *zap.Su
 	logger = logger.With(zap.String("controller/impl", "pkg"))
 	logger.Info("Starting the controller")
 
-	const numControllers = 4
+	const numControllers = 5
```

Comment:
I do wonder if it would make sense to put the sources into a separate controller binary at least?

Reply:
hm, I thought it was a nice feature to host them all in one, but I am open to the idea. 

Can we start with this and split them if it is needed?

Reply:
ok doing it.

Reply:
done.